src/main/kotlin/net/atos/zac/signalering/SignaleringService.kt

```diff
@@ -223,11 +222,11 @@ class SignaleringService @Inject constructor(
         val root = query.from(Signalering::class.java)
         return entityManager.createQuery(
             query.select(root)
-                .where(getSignaleringWhere(parameters, builder, root))
+                .where(getSignaleringWhere(signaleringSearchParameters, builder, root))
                 .orderBy(builder.desc(root.get<Any>("tijdstip")))
         )
-            .setFirstResult(pageNumber * pageSize)
-            .setMaxResults(pageSize)
+            .setFirstResult(pageParameters.page * pageParameters.rows)
+            .setMaxResults(pageParameters.rows)
```

Comment:
This reads a bit strange to me; where first result is larger than max result. Why is that?

Reply:
That's how paging works:

Page 1: first = 0 * 5 = 0, max = 5
Page 2: first = 1 * 5 = 5, max =5
Page 3: first = 2 * 5 = 10, max 5